cmd/nerdctl/container_run_linux_test.go

```diff
@@ -311,6 +312,54 @@ func TestRunTTY(t *testing.T) {
 	assert.Equal(t, 0, res.ExitCode, res.Combined())
 }
 
+func TestRunSigProxyDefault(t *testing.T) {
```

Comment:
Can we condense these 3 tests into one? [This](https://github.com/containerd/nerdctl/blob/ab956d0c84d05bdee06361bc03186526d22c2faa/pkg/cmd/builder/build_test.go#L72) might be a decent example.